tests/data_flow_test.go

```diff
@@ -21,6 +21,7 @@ package tests
 import (
```

Comment:
To test smart routing in data flow we will need the ability to stand up multiple edge routers right? I know the way it is currently done stores a single instance.

Reply:
Yes and now. We can test terminator selection strategies using a single router with multiple clients. I have a test written for that which I'll put up in the next pass. But for pure smart routing we'll need multiple routers since that is done just based on router <-> router metrics. I'm not sure if we need to do that in this kind of test though. Feels like that might be more appropriate for a fablab based test.